dspace-api/src/main/java/org/dspace/access/status/AccessStatusBuilder.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.access.status;
+
+import java.sql.SQLException;
+import java.util.Date;
+
+import org.dspace.content.Item;
+import org.dspace.core.Context;
+
+/**
+ * Plugin interface for the access status calculation.
+ */
+public interface AccessStatusBuilder {
```

Comment:
Well, Builder is a well-known pattern that is used widely all over the Java landscape, not just in our test-object scaffolding, to accumulate the particulars of a complex object and then instantiate it.  But "...Builder" *is* confusing here, because this interface does not look anything like the Builder pattern.  To me it looks more like a Helper.

Reply:
I renamed it with the Helper suffix, so it's gonna be less confusing. Thanks @mwoodiupui for the suggestion.